interface/patient_file/report/custom_report.php

```diff
@@ -45,6 +45,12 @@
 $PDF_OUTPUT = empty($_POST['pdf']) ? 0 : intval($_POST['pdf']);
 
 if ($PDF_OUTPUT) {
+    /* Because of the nature of this report, it is difficult to catch all the various errors possible
+     * so, turning on E_ERROR seems logical to catch memory and php errors that can't be reported due to output buffering.
+     * Try/catch will handle errors after buffer content is retrieved.
+     */
+    ini_set('display_errors', 1); // turn on diswplaying of errors.
+    error_reporting(E_ERROR); // Just fatal php errors are what concern us.
```

Comment:
Believe it or not but try as I may, I found only two things that will break this report: 1. non utf8 characters and sending content before header. Even malformed html won't break parser as mpdf simply shows it the way it received it(yeah).Non utf8 does not exit well and runs out of memory(or so it claims)!
Bottom line is either we let user sit there scratching their head or let php do it's job and throw the error to interface. However, you would not bring up unless there is a reason so I will try to find another way.